anonymizer/OpenDiffix.Service/JsonEncodersDecoders.fs

```diff
@@ -31,6 +41,7 @@ type PreviewRequest =
     Salt: string
     Buckets: string []
     CountInput: CountInput
+    AnonParams: Option<RequestAnonParams>
```

Comment:
Like Cristian noted, we should make these required so that we don't end up with all the loose defaults around.

Reply:
ok, I'll handle this in this PR, since we know how to organize these already and I need to correct style anyway.